### PR TITLE
Add visualization scripts

### PR DESCRIPTION
**Pull request by alvin-garcia**
I still need to incorporate line-outs and handle non-axisymmetric inputs. Please review my preliminary version.

### CODE REVIEW

**Comment by alvin-garcia** [comment]
Let's wait a little bit more before merging in this branch. Plot_inputs is very user friendly, but I don't think plot_outputs is there yet. Although functional, it has a lot of optional arguments that may make a new user confused. I would still appreciate any suggestions you may have. Thanks.

**Comment by alvin-garcia** [comment]
I fixed the merge commits. Let me know what you think please.

File: src/fidasim.f90 [hunk 1]
```diff
@@ -5142,7 +5149,7 @@ end subroutine line_basis
 subroutine plane_basis(center, redge, tedge, basis, inv_basis)
     !+ Calculates basis from 3 points on a plane with +z being the plane normal
     real(Float64), dimension(3), intent(in)              :: center
-        !+ Plane origin
+        !+ Position in cylindrical coordinates
```
Comment (lstagner) [inline src/fidasim.f90]: This is not necessarily has to be in cylindrical coordinates. In fact this subroutine is used in the hit_npa_detector in which beam grid coordinates are used.

File: src/fidasim.f90 [hunk 2]
```diff
@@ -6020,6 +6139,153 @@ subroutine track(rin, vin, tracks, ntrack, los_intersect)
...
+        tedge = side_max + zstep
+        call rz_normal_vector(side_max, redge, tedge, n0)
+        call line_plane_intersect(ri, vn, side_max, n0, p, t, cyl=.True.)
+        dt_arr(4) = t
+        ! inner_circle = side_min
+        call line_circle_intersect(ri, vn, side_min, p, t)
+        dt_arr(5) = t
+        outer_circle = side_min + rstep
+        call line_circle_intersect(ri, vn, outer_circle, p, t)
+        dt_arr(6) = t
```
Comment (lstagner) [inline src/fidasim.f90]: You call `line_plane/circle_intersect` many times and each time you are doing the same conversion back to cylindrical coordinates. This is inefficient. I would just use `uvw` coordinates throughout making it simpler and more efficient.

File: src/fidasim.f90 [hunk 3]
```diff
@@ -1114,6 +1119,8 @@ module libfida
     !+ Variable for storing the calculated passive NPA results
 type(SpectralChords), save      :: spec_chords
     !+ Variable containing the spectral system definition
+type(SpectralChords), save      :: igrid_spec_chords
+    !+ Variable containing the spectral system definition
```
Comment (lstagner) [inline src/fidasim.f90]: I would just add a few new fields to the `SpectralChords` type. You also haven't calculated the line of sight intersections for the cylindrical grid. This should be done in the `read_chords` subroutine. Also what happens if the user doesn't provide a `phi` dependence will the passive/cold calculations just not work?

**Comment by lstagner** [comment]
What is the status of this?

**Pull request closed without merging**